package.json

```diff
@@ -115,20 +110,13 @@
         "@babel/cli": "^7.7.7",
         "@babel/core": "^7.9.6",
         "@babel/node": "^7.7.7",
-        "@babel/preset-env": "^7.9.6",
-        "@babel/runtime": "^7.9.6",
-        "@babel/runtime-corejs2": "^7.9.6",
         "@storybook/addon-actions": "^6.3.4",
         "@storybook/addon-console": "^1.2.3",
-        "@storybook/addon-essentials": "^6.3.4",
```

Comment:
I think we still need this one.  Storybook still launched, but I got this error in the console: 
```
ERR! Addon value should end in /register OR it should be a valid preset https://storybook.js.org/docs/react/addons/writing-presets/
ERR! @storybook/addon-essentials
```

Also, when I ran `npm ci` and then ran Storybook, the `Controls` tab where all the toggles were located was missing. Only the `Actions` tab was available, so might be related.

Reply:
I missed checking the `Controls` tab. Good Catch 👍🏽